test/Game-test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Game = require('../js/Game.js');
+const spies = require('chai-spies');
+chai.use(spies);
+global.data = require('../js/data.js');
+global.domUpdates = require('../js/DOM.js');
+global.Round = require('../js/Round.js');
+// global.index = require('../js/index.js');
+chai.spy.on(global.domUpdates, ['getPlayerNames', 'clearInputs', 'goToGameScreen', 'displayWinner', 'goToHomeScreen', 'displayWheel', 'hideWheel'], () => true);
+
+describe('Game', () => {
+  var game;
+  beforeEach(() => {
+    game = new Game()
+  })
+
+  it('should start at round 0', () => {
+  expect(game.round).to.equal(0);
+  expect(game.bonusRound).to.equal(false);
+  });
+
+  it('should start with no players', () => {
+  expect(game.players).to.deep.equal({});
+  });
+
+  it('should start with 4 puzzle banks', () => {
+    expect(game.puzzleKeys).to.deep.equal(['one_word_answers', 'two_word_answers', 'three_word_answers', 'four_word_answers']);
+  });
+
+  it.skip('should be able to take in player names', () => {
+   
+  });
+
+  it('should start a new game', () => {
+    game.init();
+    expect(global.domUpdates.clearInputs).to.have.been.called(1);
+    expect(global.domUpdates.goToGameScreen).to.have.been.called(1);
+  });
```

Comment:
It description here is a little vague -- what actually happens when we start a new game? What does it mean to start a new game?  I'd wrap this it block in a describe block that says:

```js
describe('starting a new game', () => {
    it('should clear the player name inputs', () => {});
    it('should go to the game screen', () => {});
    it('should set the game players', () => {});
});